src/test/java/com/softserveinc/ita/ProtocolTest.java

```diff
@@ -1,25 +1,49 @@
 package com.softserveinc.ita;
 
-import static com.codeborne.selenide.Selenide.webdriver;
-import static org.assertj.core.api.Assertions.assertThat;
-
 import com.softserveinc.ita.pageobjects.LoginPage;
+import com.softserveinc.ita.pageobjects.admin.ProtocolPage;
 import com.softserveinc.ita.pageobjects.util.TestRunner;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
 
+import static com.codeborne.selenide.Selenide.webdriver;
+import static com.softserveinc.ita.pageobjects.util.DataProvider.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
 public class ProtocolTest extends TestRunner {
 
-    @Test
-    public void verifyProtocolPageOpening() {
-        // TODO: 13.05.2022 use config files for credentials and expected url
+    @BeforeMethod
+    public ProtocolPage openProtocolPage() {
```

Comment:
in test classes, methods should not return anything, if you want to init ProtocolPage better to create a class variable

```
private ProtocolPage protocolPage;

@BeforeMethod
    public void openProtocolPage() {
        protocolPage = new LoginPage()
                .login(ADMIN_LOGIN, ADMIN_PASSWORD)
                .openProtocolPage();
    }
```